### PR TITLE
Add docs about brew upgrade HZ-618

### PR DESCRIPTION
**Pull request by ldziedziul**
Related to https://hazelcast.atlassian.net/browse/HZ-618

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,74 @@
-# Homebrew Tap Repository for Hazelcast Command Line
+# Homebrew Tap Repository for Hazelcast Platform
 
-`hz` is a tool which allows users to install & run [Hazelcast IMDG](https://hazelcast.org/imdg/) and [Management Center](https://hazelcast.org/imdg/download/#hazelcast-imdg-management-center) on local environment.
+`hz` is a tool which allows users to install & run [Hazelcast IMDG](https://hazelcast.org/imdg/) on a local environment.
```
Comment (JakeSCahill) [inline README.md]: IMDG?

File: README.md [hunk 2]
```diff
@@ -1,54 +1,74 @@
...
+Install particular minor version of Hazelcast
+
+    brew install hazelcast-5.1
 
 Install particular version of Hazelcast
 
-    $ brew install hazelcast@5.0.1
+    brew install hazelcast@5.0.1
+
+NOTE: Upgrade of a particular version is not supported by brew. You need to uninstall the old package first
```
Comment (JakeSCahill) [inline README.md]:
```suggestion
NOTE: To upgrade to a particular version, uninstall the old package first 
```

File: README.md [hunk 3]
```diff
@@ -1,54 +1,74 @@
...
+
+    brew install hazelcast-5.1
 
 Install particular version of Hazelcast
 
-    $ brew install hazelcast@5.0.1
+    brew install hazelcast@5.0.1
+
+NOTE: Upgrade of a particular version is not supported by brew. You need to uninstall the old package first 
+and then install `hazelcast` or `hazelcast-x.y` instead
```
Comment (JakeSCahill) [inline README.md]:
```suggestion
and then install `hazelcast` for the latest version or `hazelcast-x.y` for another version.
```

File: README.md [hunk 4]
```diff
@@ -1,54 +1,74 @@
...
-    $ brew install hzcloud 
+    brew install hzcloud
 
 ### To clean up everything (may come in handy):
 
-    $ brew uninstall hazelcast
-    $ brew uninstall hzcloud
-    $ brew untap hazelcast/hz
-    $ brew cleanup -s
+    brew uninstall hazelcast
```
Comment (JakeSCahill) [inline README.md:67]: Should we add `\` to the end of these lines so that copy/paste works?

File: README.md [hunk 5]
```diff
@@ -1,54 +1,74 @@
...
 
 Install without Java dependency (you already have installed Java in 
 another way)
 
-    $ brew install hazelcast --without-openjdk
+    brew install hazelcast --without-openjdk
+
+### Configuration
+
+Starting from the 5.1 version configuration files are preserved between upgrades and stored in
```
Comment (JakeSCahill) [inline README.md]:
```suggestion
Starting from version 5.1, configuration files are preserved between upgrades and stored in
```

File: README.md [hunk 6]
```diff
@@ -1,54 +1,74 @@
...
 
-See [Hazelcast Command Line](https://github.com/hazelcast/hazelcast-command-line/) for the source code.
+See [Hazelcast Packaging](https://github.com/hazelcast/hazelcast-packaging) for the source code.
 
 See [Hazelcast Cloud Command Line](https://github.com/hazelcast/hazelcast-cloud-cli) for the source code
 
 ## How to install
 
-### Tap repository
+### Add tap repository
```
Comment (JakeSCahill) [inline README.md]:
```suggestion
### Add Tap Repository
```

File: README.md [hunk 7]
```diff
@@ -1,54 +1,74 @@
...
+    $(brew --prefix)/etc/hazelcast
+
+(usually `/usr/local/etc/hazelcast` or `/opt/homebrew/etc/hazelcast`)
 
 ### Install Hazelcast CLoud CLI
 
-    $ brew install hzcloud 
+    brew install hzcloud
 
 ### To clean up everything (may come in handy):
```
Comment (JakeSCahill) [inline README.md]:
```suggestion
### Clean Up
```

File: README.md [hunk 8]
```diff
@@ -1,54 +1,74 @@
...
-`hz` is a tool which allows users to install & run [Hazelcast IMDG](https://hazelcast.org/imdg/) and [Management Center](https://hazelcast.org/imdg/download/#hazelcast-imdg-management-center) on local environment.
+`hz` is a tool which allows users to install & run [Hazelcast IMDG](https://hazelcast.org/imdg/) on a local environment.
 
-`hzcloud` is a command line tool to make actions on Hazelcast Cloud easily
+`hzcloud` is a command line tool to make actions on Hazelcast Cloud easily.
 
 ## About
 
-This repository contains the Homebrew formula for `hz` and  `hzcloud`.
+This repository contains the Homebrew formulas for `hazelcast`, `hazelcast-enterprise` and  `hzcloud`.
```
Comment (JakeSCahill) [inline README.md]: I think this should be the first sentence before we start explaining what `hz` and `hc-cli` are.

We should also provide a brief description of each one and then describe what CLI tools come with each (`hz` or `hzcloud`).

File: README.md [hunk 9]
```diff
@@ -1,54 +1,74 @@
-# Homebrew Tap Repository for Hazelcast Command Line
+# Homebrew Tap Repository for Hazelcast Platform
 
-`hz` is a tool which allows users to install & run [Hazelcast IMDG](https://hazelcast.org/imdg/) and [Management Center](https://hazelcast.org/imdg/download/#hazelcast-imdg-management-center) on local environment.
+`hz` is a tool which allows users to install & run [Hazelcast IMDG](https://hazelcast.org/imdg/) on a local environment.
 
-`hzcloud` is a command line tool to make actions on Hazelcast Cloud easily
+`hzcloud` is a command line tool to make actions on Hazelcast Cloud easily.
```
Comment (JakeSCahill) [inline README.md]: 'Make actions' isn't helpful.

Maybe 'for managing Hazelcast Cloud clusters'?

**Review by JakeSCahill — COMMENTED** [review]
Some general feedback/questions.

File: README.md (hunk 1 above)
Comment (ldziedziul) [reply README.md]: hmm, I don't follow, should I use `Hazelcast Platform` instead?

File: README.md (hunk 4 above)
Comment (ldziedziul) [reply README.md:67]: They're all non-interactive commands so new lines should be fine

File: README.md (hunk 1 above)
Comment (ldziedziul) [reply README.md]: changed: https://github.com/hazelcast/homebrew-hz/pull/27/commits/138cc6e9f9dc3a74f033fb613c85fa1e881c1449

File: README.md (hunk 8 above)
Comment (ldziedziul) [reply README.md]: Changed: https://github.com/hazelcast/homebrew-hz/pull/27/commits/138cc6e9f9dc3a74f033fb613c85fa1e881c1449

File: README.md (hunk 9 above)
Comment (ldziedziul) [reply README.md]: Changed: https://github.com/hazelcast/homebrew-hz/pull/27/commits/138cc6e9f9dc3a74f033fb613c85fa1e881c1449

**Comment by ldziedziul** [comment]
> Some general feedback/questions.

Many thanks for the review, I think I've responded to all of them

**Review by JakeSCahill — APPROVED** [review]
Nice improvements 😄

**Pull request merged**